### ISSUE TITLE
Ability to set Preview Background colour

### ISSUE THREAD

**Issue by kevinoneill** [AvaloniaUI/Avalonia#13662]
I find it challenging to visualise how my UIControls will look in context as they often rely on the parent to set the background.

I would like the preview plugin to include a feature that allows users to change the background colour within the preview. This would enable a more accurate and flexible design process, allowing for instant visualisation of how UI components interact with different background colours.

As an alternative, I have tried manually setting the background colour in the code and refreshing the preview, but this is a cumbersome and time-consuming process.

**Comment by timunie** [comment]
Yeah I agree this would be useful to have. I think we need these two properties: 

```diff
+ d:DesignBackground
+ d:DesignRequestedThemeVariant
```

Any help is appreciated if someone wants to implement this feature.

**Comment by Veritera** [comment]
bump
